sphinx/util/docutils.py

```diff
@@ -32,8 +31,6 @@
     from sphinx.config import Config
     from sphinx.environment import BuildEnvironment
 
-
-__version_info__ = version.parse(docutils.__version__).release
```

Comment:
It's better to emit a warning on accessing this attribute instead of removal. Could you revert this change, please?